Index: src/test/java/tests/uiCheckTests/CheckTheAvailabilityOfTheMainElements.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/tests/uiCheckTests/CheckTheAvailabilityOfTheMainElements.java b/src/test/java/tests/uiCheckTests/CheckTheAvailabilityOfTheMainElements.java
new file mode 100644
--- /dev/null	
+++ b/src/test/java/tests/uiCheckTests/CheckTheAvailabilityOfTheMainElements.java	
@@ -0,0 +1,30 @@
+package tests.uiCheckTests;
+
+import api.clients.ApiBoardClient;
+import api.pojo.requests.BoardBuilder;
+import com.codeborne.selenide.Selenide;
+import io.qameta.allure.Description;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import tests.TestInit;
+
+public class CheckTheAvailabilityOfTheMainElements extends TestInit {
+    private ApiBoardClient apiBoardClient = new ApiBoardClient(BASE_URL);
+    private BoardBuilder boardBody;
+    private SoftAssert softAssert = new SoftAssert();
+
+    @BeforeMethod
+    private void setUp() {
+        boardBody = BoardBuilder.builder().build();
+    }
+
+    @Test(description = "PJ2024-35")
+    @Description("TC Checking the creation of a new board")
+    public void test() {
+        apiBoardClient.createNewBoard(boardBody);
+        Selenide.refresh();
+
+    }
+
+}
